### DISCUSSION TITLE
Thread affinity

### DISCUSSION THREAD

**Discussion by yanmdf** [PurpurMC/Purpur#1154] [__Dismissed__] [answered]
Hello good afternoon I would like to request a function called thread affinity, some time ago I noticed this function in a fork called WindSpigot that uses the Java Thread Affinity technology, where in simple terms it isolates a core to the server, this has greatly improved performance in tests in WindSpigot, and I would like to know if it is possible to add this in Purpur

Links:
WindSpigot: https://github.com/Wind-Development/WindSpigot
Java Thread Affinity: https://github.com/OpenHFT/Java-Thread-Affinity

**Comment by emielderckx** [comment, marked as answer]
Purpur is mainly a feature-oriented fork, the great performance you get with Purpur are from Paper and Pufferfish. 

You could suggest this to them, but we don't think this will be useful since Minecraft is not using a single thread any more and so this could actually hurt server performance.
